components/Splitter.scss

```diff
@@ -3,10 +3,13 @@
   top: 0;
   right: 0;
   z-index: 1;
-  cursor: pointer;
+  transition: border-radius var(--transition-duration);
+  cursor: default;
```

Comment:
Parts of the UI use a pointer, parts use the regular cursor. I think we should standardize it and use the default, the buttons already have a hover state so they don't need `pointer`:

https://github.com/tailwindlabs/tailwindcss/issues/8961#issuecomment-1533249212